roles/openshift_version/tasks/main.yml

```diff
@@ -48,17 +78,35 @@
   include: set_version_containerized.yml
   when: is_containerized | bool
 
+# Stop if the user has provided an openshift_image_tag but is not doing a containerized install
+- assert:
+    that: openshift_image_tag is not defined
+    message: >
+      openshift_image_tag is used for containerized installs. If you are trying to
+      specify an image for a non-container install see oreg_url.
+  when: not is_containerized | bool
```

Comment:
This does look like an issue when we get to mixed container + rpm installations, which are probably broken right now anyhow but it has been discussed as a desirable end state and I know there are open bugs to get it working.